charts/spark-operator-chart/templates/pod-templates-config-map.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Are we mounting these through a config map? If so, why are we adding this to the operator itself instead of our own repo? 

I want to make sure we're taking the right steps to make the fork easily maintainable.

Reply:
In the latest version of the operator, you can add in your own volumes and mounts https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/charts/spark-operator-chart/templates/deployment.yaml#L100C1-L101C1

Maybe we should just update the fork?

Reply:
these pod template files need to be mounted on the operator pod which initiates the driver/executor pod creation. Whenever the driver/executor pod is initiated, the pod template files should be accessible from the operator pod (whether they're in local/s3/gcs).

Check https://spark.apache.org/docs/latest/running-on-kubernetes.html#pod-template for more details.

Reply:
yes. it makes sense to update the fork instead of recreating it. i'll look into this update.

Reply:
Yes I read through the links provided. Putting the files on the operator image does not make them accessible to the driver or executor. It's the config map, right? Take a look at the code for how they expect users to add their own volumes.

Reply:
the pod template files are not required to be accessible by driver/executor. these files are required to create driver & executor pods. That's why it should be accessible on the operator pod. i've tested this flow in gcp-acme org, in staging namespace. Can you please take a look? I've updated [the PR](https://github.com/onehouseinc/gateway-controller/pull/3487) on gateway-controller to create create the volume from the latest version of the operator itself. This PR is no longer needed.